api/.env.example

```
@@ -17,3 +17,15 @@ JWT_SECRET_KEY=
+
+# ==========================================
+# MONOBANK
+# ==========================================
+MONOBANK_API_TOKEN=
+MONOBANK_BASE_URL=
+
+# ==========================================
+# TEMP FRONTEND/BACKEND URL
+# ==========================================
+FRONTEND_URL_BASE=
+BACKEND_URL_BASE=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BACKEND_URL_BASE key should go before the FRONTEND_URL_BASE key

(UnorderedKey)